src/rpc/rawtransaction.cpp

```diff
@@ -207,25 +208,34 @@ PartiallySignedTransaction ProcessPSBT(const std::string& psbt_string, const std
     }
 
     // If we still haven't found all of the inputs, look for the missing ones in the utxo set
-    if (!coins.empty()) FindCoins(node, coins);
+    if (!coins.empty()) {
+        FindCoins(node, coins);
+        for (unsigned int i = 0; i < psbtx.tx->vin.size(); ++i) {
+            PSBTInput& input = psbtx.inputs.at(i);
+
+            // If there are still missing utxos, add them if they were found in the utxo set
+            if (!input.non_witness_utxo) {
+                const CTxIn& tx_in = psbtx.tx->vin.at(i);
+                const Coin& coin = coins.at(tx_in.prevout);
+                if (IsSegWitOutput(provider, coin.out.scriptPubKey) && !coin.out.IsNull()) {
+                    input.witness_utxo = coin.out;
+                }
+            }
+        }
+    }
+
+    const PrecomputedTransactionData& txdata = PrecomputePSBTData(psbtx);
```

Comment:
In 2b67f9b73143e07a30add5f372395a612762da46 "rpc: add descriptorprocesspsbt rpc"

Seems like this should be in the previous commit.

Reply:
I've moved this to the previous commit.